customTests/csc_nvidia_smi.nhc

```diff
@@ -38,35 +38,35 @@ function check_nvsmi_healthmon() {
         pass 0 "$FUNCNAME: $NVIDIA_SMI_HEALTHMON completed successfully"
         return 0
     elif [[ $NVSMI_HEALTHMON_RC -eq 4 ]]; then
-        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Permission denied"
+        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Permission denied. FaultCode: NHCNA"
         return 1
     elif [[ $NVSMI_HEALTHMON_RC -eq 8 ]]; then
-        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Power cables not attached"
+        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Power cables not attached. FaultCode: NHC2001"
         return 1
     elif [[ $NVSMI_HEALTHMON_RC -eq 2 ]]; then
-        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Invalid argument or flag"
+        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  Invalid argument or flag. FaultCode: NHCNA"
         return 1
     elif [[ $NVSMI_HEALTHMON_RC -eq 9 ]]; then
-        die 1 "$FUNCNAME:  $NVIDIA_SMI_HEALTHMON:  NVIDIA driver not loaded"
```

Comment:
Should these be different fault codes instead of "NHCNA" since the errors are different?

Reply:
NHCNA, is used when there is not a specific fail case in ImpactRP. It could be a place holder for the future or indicate that this is not an actual hardware issue but an environment issue.